packages/nx-plugin/src/generators/configure-storybook/files/.babelrc.template

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../babel.config.json"
```

Comment:
this shouldn't be here as that has nothing to do with storybook, this nicely shows that we need library generator for react and node

Reply:
I added this here, as it's failing on `storybook-build` command... what do you suggest?

Reply:
right, yeah I know why you added it there. can you create issue to introduce a `react` library generator ? that's where some of the "misused" logic of storybook config generator belongs

Reply:
https://github.com/microsoft/fluentui-contrib/issues/93

Reply:
yes , thanks !